README.md

```diff
@@ -72,8 +72,17 @@ The project depends on a SQLite database. This is not yet initialized and is the
 
 The website should now be in working order on your computer. On running the website with the Play button in PyCharm (or command if not using PyCharm), you should no longer see an error message.
 
+### Creating new pages.
+
+New pages can be created by going to localhost:8000/admin -> Pages -> click on the pencil next to the title -> click on the triple dots on the top of the page -> Add child page
+You can navigate to the page that you created by using the title of your page like this: localhost:8000/PAGE_TITLE after you publish using the drop up menu at the bottom of the page.
```

Comment:
If you wrap these links like this: `[localhost:8000/admin](http://localhost:8000/admin)`, they will highlight and become clickable when reading the document

The syntax is: `[friendly name](actual URL)`

Reply:
Nice, thanks for the info, I'll incorporate this 🙌